tests/scaletest/lib/ingress.go

```diff
@@ -36,6 +37,7 @@ var ingressResource = schema.GroupVersionResource{Group: "extensions", Version:
 var kubeClient dynamic.Interface
 var coreV1Client corev1.CoreV1Interface
 var appsV1Client appsv1.AppsV1Interface
+var ctx context.Context
```

Comment:
nice. Does this get initialized as context.TODO()

Reply:
My bad! It was a mistake, have changed it context.TODO() now.

Reply:
oh ok